src/native/external/libunwind.cmake

```
@@ -96,7 +96,7 @@ elseif(CLR_CMAKE_TARGET_FREEBSD)
     set(libunwind_la_SOURCES_arm_os             arm/Gos-freebsd.c)
     set(libunwind_la_SOURCES_arm_os_local       arm/Los-freebsd.c)
     set(libunwind_la_SOURCES_aarch64_os         aarch64/Gos-freebsd.c)
-    set(libunwind_la_SOURCES_aarch64_os_local   aarch64/Los-freebsd.c)
+    set(libunwind_la_SOURCES_aarch64_os_local   aarch64/Los-freebsd.c aarch64/setcontext.S)
```

We need both (only setcontext.S was missing)
```suggestion
    set(libunwind_la_SOURCES_aarch64_os_local   aarch64/Los-freebsd.c)
    set(libunwind_la_SOURCES_aarch64_os_local   aarch64/Los-freebsd.c aarch64/setcontext.S)
```

---

Isn't what my change `set(libunwind_la_SOURCES_aarch64_os_local aarch64/Los-freebsd.c aarch64/setcontext.S)` is doing? Am I wrong, that second `set` will overwrite the previous one? So it's just ok to add `aarch64/setcontext.S` to the existing line with set?

---

Ah, corrected the patch. Lets put them on separate lines for consistency.

---

New lines added.

---

I meant keep the existing style used in this file (one line added, zero removed).

---

Sorry I'm lost - there are few ways of doing set in this file. It's not possible to only add one line in here, as there's matching bracket on the line end when calling set, and this needs to go into the same variable which is included later?

---

The naming convention here is taken from libunwind, how they classify these files. Source are arch specific, others are OS specific and then few are machine independent (`mi`). Using the existing convention, that will become https://github.com/dotnet/runtime/compare/main...am11:runtime:patch-24.

Two lines because this is the first assembly file in aarch64 group. :)

---

Ok, wasn't feeling good last days, now should be ok..